### PR TITLE
refactor: fix lint error

### PR DESCRIPTION
**Pull request by masaaania**
- fix & suppress lint error

```
$ golangci-lint run ./cmd/... ./pkg/... ./hack/... ./test/...

pkg/autoops/command/webhook.go:122: line is 133 characters (lll)
func (h *webhookCommandHandler) ChangeWebhookDescription(ctx context.Context, cmd *autoopspb.ChangeWebhookDescriptionCommand) error {
pkg/auth/api/api.go:326:23: func `(*authService).checkRole` is unused (unused)
func (s *authService) checkRole(
                      ^
pkg/auth/api/api.go:299:23: func `(*authService).checkAdminRole` is unused (unused)
func (s *authService) checkAdminRole(ctx context.Context) (*eventproto.Editor, error) {
                      ^
pkg/autoops/api/error.go:272:2: `errWebhookClauseConditionValueRequiredJaJP` is unused (deadcode)
        errWebhookClauseConditionValueRequiredJaJP = status.MustWithDetails(
        ^
pkg/autoops/api/api.go:206: line is 128 characters (lll)
        if len(req.Command.OpsEventRateClauses) == 0 && len(req.Command.DatetimeClauses) == 0 && len(req.Command.WebhookClauses) == 0 {
pkg/locale/locale.go:24:2: `langs` is unused (structcheck)
        langs  []string
        ^
pkg/domainevent/domain/event.go:80:13: copylocks: assignment copies lock value to options: github.com/bucketeer-io/bucketeer/proto/event/domain.Options contains google.golang.org/protobuf/internal/impl.MessageState contains sync.Mutex (govet)
        options := defaultOptions
                   ^

```

### CODE REVIEW

File: pkg/autoops/command/webhook.go [hunk 1]
```diff
@@ -119,7 +119,9 @@ func (h *webhookCommandHandler) ChangeWebhookName(ctx context.Context, cmd *auto
 	return h.publisher.Publish(ctx, event)
 }
 
-func (h *webhookCommandHandler) ChangeWebhookDescription(ctx context.Context, cmd *autoopspb.ChangeWebhookDescriptionCommand) error {
+func (h *webhookCommandHandler) ChangeWebhookDescription(
+	ctx context.Context,
+	cmd *autoopspb.ChangeWebhookDescriptionCommand) error {
```
Comment (cre8ivejp) [inline pkg/autoops/command/webhook.go:124]: Could you please refactor it as follows as we do in other files to keep consistency?

E.g.

```go
func (h *webhookCommandHandler) ChangeWebhookDescription(
    ctx context.Context,
    cmd *autoopspb.ChangeWebhookDescriptionCommand,
) error {
    ....
}
```

Comment (masaaania) [reply pkg/autoops/command/webhook.go]: I fixed it, thank you.

File: pkg/auth/api/api.go [hunk 2]
```diff
@@ -295,75 +293,3 @@ func (s *authService) maybeCheckEmail(ctx context.Context, email string) error {
 	)
 	return localizedError(statusAccessDeniedEmail, locale.JaJP)
 }
-
```
Comment (ono-max) [inline pkg/auth/api/api.go:298]: I am curious as to why we need to remove this code.

Comment (masaaania) [reply pkg/auth/api/api.go:298]: `checkAdminRole` and `checkRole` in `pkg/auth/api/api.go` are not used, so golangci-lint pointed out it.

These same codes exist and used in many files like `pkg/account/api/admin_account.go` and `pkg/environment/api/api.go`.

**Review by cre8ivejp — APPROVED** [review]
Thank you!!

File: pkg/auth/api/api.go (hunk 2 above)
Comment (ono-max) [reply pkg/auth/api/api.go:298]: I see, thank you!

**Review by ono-max — APPROVED** [review]

**Pull request merged**